server/flights.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"chai/database/sqlc"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func (a *App) FlightsHandler(w http.ResponseWriter, r *http.Request) {
+	depAirport := r.URL.Query().Get("departure_airport")
+	arrAirport := r.URL.Query().Get("arrival_airport")
+	if depAirport == "" || arrAirport == "" {
+		http.Error(w, "Missing Arrival or Departure Airport", http.StatusBadRequest)
+	}
+
```

Comment:
Never combine errors like this where the error can be interpreted multiple ways.

That just makes debugging harder. Just be unambigious with the error messages, it'll be a little more verbose, but worth it in the long run.